src/app/theme.ts

```diff
@@ -45,6 +65,11 @@ const color = {
       darkest: '#818181',
       foreground: textColors.white,
     },
+    darkGray: {base: '#818181'},
+    darkBlue: {
```

Comment:
this isn't one of the theme colors from the pattern library so we shouldn't put it here, it looks like what you want is link styles, maybe copy [linkStyle](https://github.com/openstax/rex-web/pull/81/files#diff-2dbbc782d04e2cc39b4f4bcad6fe5049R4) from the other PR